packages/backend/src/apps/telegram-bot/actions/send-message/index.ts

```diff
@@ -43,18 +43,19 @@ const action: IRawAction = {
       key: 'disableNotification',
       type: 'dropdown' as const,
       required: false,
-      value: false,
+      value: 'no',
+      showOptionValue: false,
       description:
         'Sends the message silently. Users will receive a notification with no sound.',
       variables: false,
       options: [
         {
           label: 'Yes',
-          value: true,
+          value: 'yes',
```

Comment:
hmm.. why is there a need to change this? This is a little dangerous because we're now sending `yes` / `no` to Telegram's API, instead of `true`/`false`. Although it works, it technically violates the API, since disable_notifications is typed as a boolean instead of string

Reply:
wahh thanks for the good catch, i forgot it affects the implementation, the reason is because `SingleSelect` or say dropdown should be accepting strings only after checking with Kar Rui, so if it's a boolean, the display is wonky and won't show properly when the value is `false`

suggestion: use 'yes' and 'no' but need to check if the value === 'yes' (and convert it to true boolean) and if it is a boolean already (backwards compat), just stick to using the boolean?

Reply:
As discussed, this will be replaced with `boolean-radio`